### DISCUSSION TITLE
Discord channel for voice chat

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#2749] [Announcements]
I have started a Discord channel inspired by https://github.com/pyvista/pyvista/discussions/2662.  If you begin a voice channel with video someone may find you and chat in voice.
https://discord.gg/HDfWVTR8te

**Comment by banesullivan** [comment]
Is this still live?
<img width="1013" alt="Screen Shot 2022-06-21 at 3 49 40 PM" src="https://user-images.githubusercontent.com/22067021/174902941-78351704-2cb3-494a-aec4-573cc41c7a06.png">

**Reply by tkoyama010** [reply]
Oops, I forgot to make the link indefinite.
https://discord.gg/HDfWVTR8te

**Reply by whophil** [reply]
Thanks @tkoyama010 for setting this up!

As a general comment - seems like there is already some fragmentation of PyVista discussion between Slack and GitHub Discussions. With this, there is now a Discord, too. Would be nice to understand which is preferred and for what.

**Reply by tkoyama010** [reply]
Thank you for your comment. I think they can be categorized as follows.

1. The active PyVista community members are often on [Slack](http://slack.pyvista.org/). Therefore, I think it is the best way to share events and information. However, asking questions in Slack may not be a good idea in terms of archiving as it does not keep history in free.
1. [GitHub Discussion](https://github.com/pyvista/pyvista/discussions) is the best place to ask questions as the entire archive is kept; [Discord](https://discord.gg/HDfWVTR8te) also keeps a history, but it depends on Discord Inc's policy.
1. [Discord](https://discord.gg/HDfWVTR8te) is the primary service for free Voice Chat. It is most convenient when you want to communicate face to face.
1. [Gitter](https://gitter.im/pyvista/community) is an open-source service, so there is no risk like Slack or Discord, but it is not a major player.

Of course, there is no need to follow the above ideas anywhere. This is my personal opinion.
